ruby/red-arrow/ext/arrow/raw-records.cpp

```diff
@@ -144,6 +144,118 @@ namespace red_arrow {
       // The number of columns.
       const int n_columns_;
     };
+
+    class RawRecordsProducer : private Converter, public arrow::ArrayVisitor {
+    public:
+      explicit RawRecordsProducer()
+        : Converter(),
+          record_(Qnil) {
```

Comment:
Could you initialize all members explicitly?

```suggestion
          record_(Qnil),
          column_index_(0),
          row_offset_(0) {
```